book/src/development/adding_lints.md

```diff
@@ -612,6 +613,16 @@ output in the `stdout` part.
 
 [`dbg!`]: https://doc.rust-lang.org/std/macro.dbg.html
 
+## Opposing lints
```

Comment:
I would call this `Conflicting Lints` and rephrase the paragraph below a little, that a new lint might conflict with an existing lint, as it suggests something an existing lint would complain about and if that's the case, the 2 rules below should be applied